app/views/content/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Ben's favourite things about teaching
+title_paragraph: |-
+  <p>Ben is a science and maths teacher from Dartford.</p>
```

Comment:
we describe him as 'maths and science' on the pay and benefits page - we should be consistent either way?

Reply:
Updated this to align them.